### ISSUE TITLE
Sticky post should be always sticky

### ISSUE THREAD

**Issue by PeopleInside** [flarum/core#1647]
## Bug Report

**Current Behavior**
A clear and concise description of the behavior.

Sticky post should be always sticky. 

**Steps to Reproduce**
1. Go to https://discuss.flarum.org/
2. Change order Latest to older
The Sticky post are no more on top with the sticky icon.

**Expected Behavior**
See always sticky post on top with the relative sticky icon

**Screenshots**
![00](https://user-images.githubusercontent.com/5006150/48788170-8d2f7480-ecea-11e8-8d0b-0fa12b2d9a76.png)
![01](https://user-images.githubusercontent.com/5006150/48788185-91f42880-ecea-11e8-91b4-316bf14b7815.png)

[I have issue also in my last beta](https://community.peopleinside.it/)... when i am logged as admin the post order change... and sticly post are not more on top.

![03](https://user-images.githubusercontent.com/5006150/48788262-be0fa980-ecea-11e8-8d2b-af6a1cff3c77.png)
![04](https://user-images.githubusercontent.com/5006150/48788276-c2d45d80-ecea-11e8-83a9-f2d26bde94bd.png)

Mine Flarum version: 0.1.0-beta.7.2.

**Comment by Zeokat** [comment]
This is more a feature than a bug. Many users requested a lot of times real sticky posts but... at this moment there is no chance to reach your expected behavior (only [modifying core](https://discuss.flarum.org/d/133-super-sticky/43) of course).

https://discuss.flarum.org/d/133-super-sticky
https://discuss.flarum.org/d/5552-how-to-add-global-sticky-to-discussions
https://discuss.flarum.org/d/3316-sticky-topics
https://discuss.flarum.org/d/2447-stickys-not-sticky
etc...

**Comment by PeopleInside** [comment]
Sticky post is not a sticky post.. For me seems to be a bug as never found a forum where sticky post are not sticky and can go down. This looks like a bug from visitor side also from my side when i log as admin and see sticky post be down on footer :) Sad to know this is considered normal. Thanks for the information.

**Comment by jordanjay29** [comment]
Indeed, this is a design choice not a bug. As @Zeokat noted, a feature request has been made. Any member of the community is welcome to make an extension to address it.

**Comment by PeopleInside** [comment]
No one has developed a working extension for a super sticky topic.
Seems i am not the only user that are having issue with current sticky works..

When i tried the last extension present in the forum https://discuss.flarum.org/d/18713-super-sticky-posts i broke my forum. Install extension made from new user are not really safe i believe...

I tried to edit the code https://discuss.flarum.org/d/18713-super-sticky-posts/17 because i need fix this sticky issue on my community... but seems if a new post are created the sticky get again moved in the second line... this seems done because the new discussion are inserted by Javascript that is not requiring updated listed by API so seems the only fix to the PHP file doesn't solve the issue.

I am very sad, seems i am forced to keep the not sticky function.. i asked for an extension in the forum but my topic was closed https://discuss.flarum.org/d/18829-super-sticky-extension-request redirected to another without a solution.

Flarum is amazing, sad to cannot solve the sticky issue that affect no only me.
